theme/js/components/search/suggest-results.vue

```diff
@@ -64,9 +62,7 @@ Used by the suggest feature to display typeahead-style results when you type you
         <span v-html="arrow" />
       </a>
       <p class="text-grey-300 m-0">
-        Nullam neque bibendum convallis enim aliquam. Integer egestas accumsan,
-        in varius lectus. Elementum a facilisis nibh pellentesque enim egestas
-        porta.
+        {{ $t('Reuses refer to the use of data for an other purpose than the one they were produced for.') }}
```

Comment:
That is a very good explanation but perhaps not pertinent to the view?

data.gov.ie says "Datasets have been used to build apps, websites and visualizations. They've been featured in articles, and written about in news reports and blog posts. Showcases collect together the best examples of datasets in use, to provide further insight, ideas, and inspiration."

Canada says "One of the benefits of Open Data is the ability to create applications. On this page you will find mobile and web-based applications, created within the Government of Canada, the public, and Canadian Open Data Experience (CODE) winners and participants"

Reply:
How about the following ?
> Public datasets are constantly reused in applications, visualizations, articles, etc.. Some of these reuses are referenced here.

Reply:
Awesome!